src/main/java/cs/f10/t1/nursetraverse/logic/commands/appointment/FindAppointmentCommand.java

```diff
@@ -32,7 +32,7 @@ public CommandResult execute(Model model) {
         requireNonNull(model);
         model.updateFilteredAppointmentList(predicate);
         return new CommandResult(
-                String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
+                String.format(Messages.MESSAGE_PATIENTS_LISTED_OVERVIEW, model.getFilteredPatientList().size()));
```

Comment:
Same issue, can import this above.

I am just skimming through the changes so I am not too sure where else this might have happened. I will try to point out in this review where else this might have happened.